src/pages/RepoPage/CoverageTab/subroute/FileExplorer/FileListTable/FileListTable.spec.tsx

```diff
@@ -571,7 +560,6 @@ describe('FileListTable', () => {
           const { requestFilters, user } = setup({})
           render(<FileListTable />, { wrapper: wrapper() })
 
-          expect(await screen.findByText('Coverage %')).toBeTruthy()
           const coverage = screen.getByText('Coverage %')
```

Comment:
nit: difference between getByText and findByText for this one?

Reply:
Ahh I just missed one instance. Good catch